src/open_inwoner/accounts/views/profile.py

```diff
@@ -63,6 +63,10 @@ def get_context_data(self, **kwargs):
         else:
             context["contact_text"] = _("U heeft nog geen contacten.")
         context["questionnaire_exists"] = QuestionnaireStep.objects.exists()
+        context["backend_auth_is_email"] = (
```

Comment:
The naming seems a bit weird to me. It is not clear what "backend_auth_is_email is doing. A better name would be "can_change_password"

Also all we want here is to see if the user is login_type != digid.